projects/sandbox/src/app/record-report/record-report.component.html

```diff
@@ -201,10 +201,21 @@
           [href]="getDatasetId() | formatHarvestUrl: summary.europeanaRecordId"
         ></sb-copyable-link-item>
       </li>
+      <li>
+        <ng-template #labelRefPublish
```

Comment:
is there a reason to write the code like this?
```
        <ng-template #labelRefPublish
          ><span i18n="record link link@@recordLinkPublish"
            >record after processing (as published)</span
          ></ng-template
        >
        <sb-copyable-link-item
          [labelRef]="labelRefPublish"
          [href]="getDatasetId() | formatHarvestUrl: summary.europeanaRecordId:'PUBLISH'"
        ></sb-copyable-link-item>
```
why not format something like this?
```
        <ng-template #labelRefPublish>
            <span i18n="record link link@@recordLinkPublish">record after processing (as published)</span>
        </ng-template>
        <sb-copyable-link-item
          [labelRef]="labelRefPublish"
          [href]="getDatasetId() | formatHarvestUrl: summary.europeanaRecordId:'PUBLISH'">
        </sb-copyable-link-item>
```

Reply:
The formatting is decided by the `prettier` plugin!